### PR TITLE
Update Travis build to add unit tests for ppc64le

### PR DESCRIPTION
**Pull request by Abhijit-Mane**
PR to enable unit tests for ppc64le in .travis.yml
Signed-off-by: Abhijit Mane <abhijman@in.ibm.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     os: linux
     arch: ppc64le
     jdk: openjdk11
-    script: mvn install -DskipTests -Dmaven.javadoc.skip=true
+    script: mvn install -Dmaven.javadoc.skip=true
```
Comment (scholzj) [inline .travis.yml:11]: Does it still need to skip the JavaDoc?

**Review by ppatierno — APPROVED** [review]
Nothing to add to the Jakub's comment.

File: .travis.yml (hunk 1 above)
Comment (Abhijit-Mane) [reply .travis.yml:11]: Yes, was facing some issues with javadoc.skip flag enabled. Will revert once it is resolved.

**Review by scholzj — APPROVED** [review]

**Comment by scholzj** [comment]
Thanks for the PR.

**Pull request merged**